src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+public class Deadline extends Task{
```

Comment:
This is minor but remember to add a space before the opening brace for the Deadline class